giga_connectome/postprocess.py

```diff
@@ -80,21 +85,14 @@ def run_postprocessing_dataset(
         Smoothing kernel size, passed to nilearn masker.
 
     output_path:
-        Full path to output file, named in the following format:
-        output_dir / atlas-<atlas>_desc-<strategy_name>.h5
+        Full path to output directory.
 
     analysis_level : str
         Level of analysis, only "participant" is available.
 
     calculate_average_correlation : bool
         Whether to calculate average correlation within each parcel.
     """
-    if analysis_level != "participant":
-        raise Warning(
-            "Only participant level analysis is supported. Other levels "
-            "would be ignored, and will run the participant level."
-        )
```

Comment:
realizing I just completely removed this from the code

but I feel that given that this is group level is not allowed by the CLI this warning will never be hit in practice

Reply:
yeah that makes sense